modules/ding_webtrekk/ding_webtrekk.module

```diff
@@ -51,24 +46,92 @@ function ding_webtrekk_page_alter(&$page) {
         'option: {}
       };' . $tag_integration_logic,
       'type' => 'inline',
-      'scope' => 'footer',
+      'group' => JS_LIBRARY,
+      'weight' => -21,
```

Comment:
Please add a comment about why this weight is set as it is. It is often in reference to a weight set by another element.

Reply:
Will do

Reply:
Just wanted to put it as high as possible in the head and [jQuery seems to have the lowest weight of system libraries](https://api.drupal.org/api/drupal/modules%21system%21system.module/function/system_library/7.x)

I can see the comment is misleading. I'll change it.